packages/client/src/config.schema.json

```diff
@@ -172,6 +172,9 @@
                             },
                             "password": {
                                 "type": "string"
+                            },
+                            "tcp": {
+                                "type": "boolean"
                             }
```

Comment:
```suggestion
                            "tcp": {
                                "type": "boolean",
                                "default": false
                            }
```

Reply:
This would add `tpc: false` to all configs (STUN only included) which is not strictly required. ie:
` {
              "port": 5349,
              "url": "stun:stun.streamr.network",
               "tcp": false
 }
`

Reply:
Ah I see, keep it as is then.